components/About/AboutPage.js

```diff
@@ -21,137 +21,135 @@ const {
-                            <div className="animation-circle" id="loader2" />
-                            <span id="loader-text">{NINTY}</span>
-                            <p>Hardwork</p>
-                        </div>
-                        <div className="animation-content">
-                            <div className="animation-circle" id="loader" />
-                            <div className="animation-circle" id="loader2" />
-                            <span id="loader-text">{NINTY}</span>
-                            <p>Passion</p>
-                        </div>
-                        <div className="animation-content">
-                            <div className="animation-circle" id="loader" />
-                            <div className="animation-circle" id="loader2" />
-                            <span id="loader-text">{NINTY}</span>
-                            <p>Dedication</p>
-                        </div>
+                <div className="animation-container">
+                    <div className="animation-content">
+                        <div className="animation-circle" id="loader" />
+                        <div className="animation-circle" id="loader2" />
+                        <span id="loader-text">{NINTY}</span>
+                        <p>Hardwork</p>
                     </div>
-                </span>
-            </div>
-            <div className="services app-content">
-                <hr />
-                <p>Testimonals</p>
-                <div className="testimonal-wrapper services-wrapper">
-                    <div className="testimonal-content">
-                        <i className="fas fa-quote-left qoute" />
-                        <p className="test">{TEST1}</p>
-                        <p className="test-name">
-                            ~ Isimeme Iwu,
-                            {' '}
-                            <i>{CLIENT}</i>
-                        </p>
+                    <div className="animation-content">
+                        <div className="animation-circle" id="loader" />
+                        <div className="animation-circle" id="loader2" />
+                        <span id="loader-text">{NINTY}</span>
+                        <p>Passion</p>
                     </div>
-                    <div className="testimonal-content">
-                        <i className="fas fa-quote-left qoute" />
-                        <p className="test">{TEST2}</p>
-                        <p className="test-name">
-                            ~ Oluwa Ayo,
-                            {' '}
-                            <i>{CLIENT}</i>
-                        </p>
+                    <div className="animation-content">
+                        <div className="animation-circle" id="loader" />
+                        <div className="animation-circle" id="loader2" />
+                        <span id="loader-text">{NINTY}</span>
+                        <p>Dedication</p>
```

Comment:
you should use an array of object and map over it to create the `animation-content`